<launch>
  <arg name="cam" default="cam1" />
  <arg name="device_id" default="0" />
  <node  name="$(arg cam)" 
         pkg="gige_cam_driver" type="camera_node.py" output="screen">
      <param name="device_id" value="$(arg device_id)"/>
      <param name="camera_manager" value="$(arg cam)"/>
      <param name="calibration_file" value="$(find gige_cam_driver)/config/camera_info_$(arg cam).yaml"/>
   </node>
</launch>